<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vehicule_inter</class>
 <widget class="QDialog" name="vehicule_inter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1441</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(211, 211, 211);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-38</y>
     <width>241</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36, 179, 122);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="ges_emp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 179, 122);
color:rgb(0, 125, 209);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Gestion Employé</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ges_mat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 179, 122);
color:rgb(0, 125, 209);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Gestion Matériels</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ges_es_vert">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 179, 122);
color:rgb(0, 125, 209);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Gestion Espaces Verts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ges_vehic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 211, 211);
color: rgb(255, 191, 62);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Gestion Véhicule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ges_appar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 179, 122);
color:rgb(0, 125, 209);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Gestion Appartements</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deconnect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 179, 122);
color: rgb(170, 0, 0);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Se Déconnecter</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>59</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="fonction">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36, 179, 122);
font: 900 12pt &quot;Franklin Gothic&quot;;
color: rgb(0, 85, 0);</string>
   </property>
   <property name="text">
    <string>Fonctionalités</string>
   </property>
  </widget>
  <widget class="QPushButton" name="accueil">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36, 179, 122);
font: 900 12pt &quot;Franklin Gothic&quot;;
color: rgb(0, 85, 0);</string>
   </property>
   <property name="text">
    <string>Accueil</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>291</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox
{
background-color: rgb(0, 125, 209);
color:  rgb(255, 191, 62);
font: 900 12pt &quot;Franklin Gothic&quot;;
border: 2px solid ;
border-radius: 12px; 
padding: 5px;
border-color: rgb(255, 191, 62);
}</string>
   </property>
   <property name="title">
    <string>Ajouter Vehicule</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(217, 217, 217);
background-color: rgb(0, 125, 209);
font: 900 12pt &quot;Franklin Gothic&quot;;
</string>
    </property>
    <property name="text">
     <string>Identifiant vehicule </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_idVehicule">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 900 9pt &quot;Franklin Gothic&quot;;
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(217, 217, 217);
background-color: rgb(0, 125, 209);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Numéro de série</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_numSerie">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 900 9pt &quot;Franklin Gothic&quot;;
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(217, 217, 217);
background-color: rgb(0, 125, 209);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Localisation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_marque">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 900 9pt &quot;Franklin Gothic&quot;;
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(217, 217, 217);
background-color: rgb(0, 125, 209);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_localisation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 900 9pt &quot;Franklin Gothic&quot;;
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(217, 217, 217);
background-color: rgb(0, 125, 209);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Marque</string>
    </property>
   </widget>
   <widget class="QPushButton" name="validerV">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 179, 122);
font: 900 12pt &quot;Franklin Gothic&quot;;
color: rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
   <widget class="QPushButton" name="annulerV">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>460</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 179, 122);
font: 900 12pt &quot;Franklin Gothic&quot;;
color: rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string>Annuler</string>
    </property>
   </widget>
   <widget class="QComboBox" name="lineEdit_type">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(217, 217, 217);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>Vehicule en question</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Voiture</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bus</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Camion</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Moto</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bicyclette</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_proprietaire">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 900 9pt &quot;Franklin Gothic&quot;;
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(217, 217, 217);
background-color: rgb(0, 125, 209);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Proprietaire</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(217, 217, 217);
background-color: rgb(0, 125, 209);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Modele</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_modele">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 900 9pt &quot;Franklin Gothic&quot;;
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(217, 217, 217);
background-color: rgb(0, 125, 209);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string> Id Employe</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxid">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 900 9pt &quot;Franklin Gothic&quot;;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>80</y>
     <width>891</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox
{
background-color: rgb(0, 125, 209);
color:  rgb(255, 191, 62);
font: 900 12pt &quot;Franklin Gothic&quot;;
border: 2px solid ;
border-radius: 12px; 
padding: 5px;
border-color: rgb(255, 191, 62);
}</string>
   </property>
   <property name="title">
    <string>Recherche</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(217, 217, 217);
background-color: rgb(0, 125, 209);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Recherche par</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recherche">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 179, 122);
font: 900 12pt &quot;Franklin Gothic&quot;;
color: rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string>Recherche</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetv">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>81</y>
      <width>861</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 900 9pt &quot;Franklin Gothic&quot;;
color: rgb(0, 0, 0);
</string>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
     <property name="font">
      <font>
       <family>Franklin Gothic</family>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>N° Série</string>
     </property>
     <property name="font">
      <font>
       <family>Franklin Gothic</family>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Localisation</string>
     </property>
     <property name="font">
      <font>
       <family>Franklin Gothic</family>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
     <property name="font">
      <font>
       <family>Franklin Gothic</family>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Marque</string>
     </property>
     <property name="font">
      <font>
       <family>Franklin Gothic</family>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Modéle</string>
     </property>
     <property name="font">
      <font>
       <family>Franklin Gothic</family>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Propriétaire</string>
     </property>
     <property name="font">
      <font>
       <family>Franklin Gothic</family>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(217, 217, 217);
background-color: rgb(0, 125, 209);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Triée par</string>
    </property>
   </widget>
   <widget class="QComboBox" name="tripar">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(217, 217, 217);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>Aucun</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Marque </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Modéle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Type</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="trier">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>40</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 179, 122);
font: 900 12pt &quot;Franklin Gothic&quot;;
color: rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string>Trier</string>
    </property>
   </widget>
   <widget class="QComboBox" name="rech_par">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(217, 217, 217);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>Aucun</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Marque </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Modéle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Type</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="rech">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 900 9pt &quot;Franklin Gothic&quot;;
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>470</y>
     <width>891</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox
{
background-color: rgb(0, 125, 209);
color:  rgb(255, 191, 62);
font: 900 12pt &quot;Franklin Gothic&quot;;
border: 2px solid ;
border-radius: 12px; 
padding: 5px;
border-color: rgb(255, 191, 62);
}</string>
   </property>
   <property name="title">
    <string>Modifier ou Supprimer</string>
   </property>
   <widget class="QPushButton" name="modifierV">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 179, 122);
font: 900 12pt &quot;Franklin Gothic&quot;;
color: rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supprimerV">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 179, 122);
font: 900 12pt &quot;Franklin Gothic&quot;;
color: rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pdf">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>201</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 179, 122);
font: 900 12pt &quot;Franklin Gothic&quot;;
color: rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string>Exportation PDF</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
